src/index.ts

```diff
@@ -26,12 +26,15 @@ const handleResult = (result: CodeOwners): void => {
     } else if (Program.format === Format.JSON) {
         console.log(JSON.stringify(result));
     } else if (Program.format === Format.PRETTY) {
-        if (result.declaration)
+        if (result.declaration) {
+            const range =
+                `[${result.declaration.from}-${result.declaration.to}] `;
```

Comment:
Very nit-picky, but imo it makes more sense to move the space at the end of the string to the line where `range` is used.